docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -1,5 +1,83 @@
 # Planning simulation
 
-!!! warning
+## Preparation
 
-    Under Construction
+1. Download and unpack a sample map.
+
+   <!--  TODO: Move to the AWF organization's Google Drive. -->
```

Comment:
@mitsudome-r Could you put the zip files here to the AWF's Google Drive, please? :pray: 
https://drive.google.com/drive/folders/1ChVJBUz82Wv-LDE7RA2hNjpmVOn2j-ZB

Reply:
We can use these links for now.https://drive.google.com/drive/folders/143ZWwDFLoVAOf81eGxpzrpmckYNKRBRe?usp=sharing

Reply:
Cool, thank you!